src/llvm-demote-float16.cpp

```diff
@@ -25,6 +25,8 @@
 #include <llvm/IR/Module.h>
 #include <llvm/IR/Verifier.h>
 #include <llvm/Support/Debug.h>
+#include "julia.h"
+#include "jitlayers.h"
```

Comment:
You may need to update the Makefile's clobbers for demote-float16 so that changing jitlayers.h will remake this file.